src/constants/links.js

```diff
@@ -0,0 +1,6 @@
+const Links = {
```

Comment:
Why wrap this with `Links` object?
We can export directly the link var:
`export const CONTRACT_URL = ....`

Reply:
I align it with the current const design. Check https://github.com/starkware-libs/cairo-playground/blob/2127c9a61bb506bef188077c07429c114b2db0e0/src/constants/action-types.js

Reply:
This is not the same, those are enums-like

Reply:
done